fcrepo-clients/src/test/java/edu/unc/lib/dl/fcrepo4/DepositRecordIT.java

```diff
@@ -172,6 +175,21 @@ public void addPremisEventsTest() throws Exception {
 		// Check that the second event has the right type
 		assertTrue(events.get(1).getResource().hasProperty(Premis.hasEventType, Premis.Ingestion));
 	}
+	
+	public void addObjectsTest() throws Exception {
```

Comment:
What would you suggest using for IDs? Or can I somehow get those atuo-generated ones into Fedora, so that it will know about them?